lib/propolis/src/lib.rs

```diff
@@ -35,3 +35,46 @@ pub mod vmm;
 
 pub use exits::{VmEntry, VmExit};
 pub use vmm::Machine;
+
+pub fn version() -> &'static str {
+    lazy_static::lazy_static! {
+        static ref VERSION: String = {
+            use std::fmt::Write;
+
+            let git = option_env!("VERGEN_GIT_BRANCH")
+                .and_then(|branch| Some((branch, option_env!("VERGEN_GIT_SHA")?)))
+                .and_then(|(branch, sha)| Some((branch, sha, option_env!("VERGEN_GIT_COMMIT_COUNT")?)));
+
+            let mut version = format!("Propolis v{}", env!("CARGO_PKG_VERSION"));
```

Comment:
Drop the Propolis prefix here? Using this with `#[clap(version)]` means you already get the prefix.

Reply:
hmm, yeah, we can do that --- my thinking was that the prefix made it obvious which was the propolis version in the SMBIOS version string, but we could also just stick the prefix in front of that specifically.